### PR TITLE
docs: Remove references to XXenableJibInit and XXenableBuildpacksInit from init docs

### PR DESCRIPTION
**Pull request by highb**
After https://github.com/GoogleContainerTools/skaffold/pull/6063 , I'm led to believe that both the buildpack and jib integrations are enabled by default so these references to enable them manually should be removed from the docs.

**Related**: #6063

**Description**

**User facing changes (remove if N/A)**
Better docs!

### CODE REVIEW

File: docs/content/en/docs/pipeline-stages/init.md [hunk 1]
```diff
@@ -41,12 +41,12 @@ If this image is one you want Skaffold to build, you'll need to manually set up
 {{</alert>}}
 
 `skaffold` init also recognizes Maven and Gradle projects, and will auto-suggest the [`jib`]({{<relref "/docs/pipeline-stages/builders#/local#jib-maven-and-gradle">}}) builder.
-Currently `jib` artifact detection is disabled by default, but can be enabled using the flag `--XXenableJibInit`.
+As of v1.27.0, `jib` artifact detection is enabled by default.
```
Comment (briandealwis) [inline docs/content/en/docs/pipeline-stages/init.md:44]: This was sufficiently long ago that we can just remove this, I think.

**Comment by briandealwis** [comment]
@highb to accept your contribution, we'll need you or your employer to sign the CLA.

**Comment by highb** [comment]
> @highb to accept your contribution, we'll need you or your employer to sign the CLA.

Yep, I'll look into getting that sorted.

File: docs/content/en/docs/pipeline-stages/init.md (hunk 1 above)
Comment (highb) [reply docs/content/en/docs/pipeline-stages/init.md:44]: Works for me, I can amend this to remove the line entirely once I've been able to sign my CLA.

**Comment by highb** [comment]
@briandealwis Signed the CLA as an individual, fixed up my commits to match Conventional Commits, and removed the line you mentioned.

**Comment by highb** [comment]
Oh ha, I guess my editor auto-corrected some whitespace issues as well.

**Review by tejal29 — APPROVED** [review]

**Pull request merged**